.changeset/config.json

```diff
@@ -4,7 +4,7 @@
   "commit": false,
   "fixed": [],
   "linked": [],
-  "access": "restricted",
+  "access": "public",
```

Comment:
This PR is an attempt to get this to return `true`

https://github.com/wp-graphql/wpgraphql-ide/blob/main/.github/workflows/release-plugin.yml#L39